### PR TITLE
LPD-26670 Implement New Naming Convention for Data Set Entities

### PR DESCRIPTION
**Pull request by markocikos**
### References

- Parent Story [LPD-6838 Update Data Set Entity naming](https://issues.liferay.com/browse/LPD-6838)
- In a way, this is a continuation of https://github.com/liferay-frontend/liferay-portal/pull/4205
- [Related Slack discussion](https://liferay.slack.com/archives/C3JBR21HA/p1716899190997769)
- Requires https://github.com/liferay-frontend/liferay-portal/pull/4227 to work correctly without FF

### What is the goal of this PR?

Overall goal is to have consistent naming everywhere based on `Data Set`. `FDS` prefix is out! This PR covers:
- endpoints
- object and relationship UI labels
- most of the frontend code, except for pre-FF legacy code. This means massive refactor, again, hopefully the last one on frontend.
- in addition to main effort, I fixed [LPD-26456 Flaky PW test: Data Set fragment configuration](https://liferay.atlassian.net/browse/LPD-26456) because it started continuously failing for me.

This PR does **not** cover:
- object entity names and classes
- fragment backend
- module/app placement
- PW tests already follow DS convention

Technical notes inline.

### What does it look like?
screen1
screen2
screen3
screen4

### CODE REVIEW

**Comment by markocikos** [comment]
ci:test:sf

File: modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java [hunk 1]
```diff
@@ -122,17 +122,17 @@ private static String _getInvokerFileNameRegex(
...
 		).put(
-			"FDSField", "/data-set-manager/fields"
+			"FDSField", "/data-set-manager/table-sections"
 		).put(
 			"FDSListSection", "/data-set-manager/list-sections"
 		).put(
 			"FDSSort", "/data-set-manager/sorts"
 		).put(
-			"FDSView", "/data-set-manager/views"
+			"FDSView", "/data-set-manager/data-sets"
```
Comment (markocikos) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java:135]: These are not persisted, so we can change them without FF. We just need to make sure both old and new UI uses  new endpoints.

File: modules/test/playwright/tests/frontend-data-set-views-web/pages/FDSFragmentPage.ts [hunk 2]
```diff
@@ -115,22 +119,6 @@ export class FDSFragmentPage {
...
 
-	async dragAndDropFragment(itemName: string) {
-		const source = await this.page.getByRole('menuitem', {
-			exact: true,
-			name: itemName,
-		});
-
-		await source.focus();
-		await source.press('Enter');
-		await source.press('Enter');
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-views-web/pages/FDSFragmentPage.ts:126]: This was causing the flaky behavior for me, I am not sure why. Manual double enter worked fine. I rewrote it to do drag & drop.

Comment (juanjofgliferay) [reply modules/test/playwright/tests/frontend-data-set-views-web/pages/FDSFragmentPage.ts:126]: Awesome. Hope that this work in a smooth way.
I remember start tinkering with the Drag & Drop but finding issues due to the side panel (widgets & fragments) hiding part of the drop area. 🤞

**Review by juanjofgliferay — APPROVED** [review]
LGTM 🚀

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by dsanz** [comment]
ci:forward

**Comment by dsanz** [comment]
ci:forward:force

**Pull request closed without merging**